src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -297,7 +304,18 @@ public override IntermediateNode Visit(TexlCallNode node, IRTranslatorContext co
                 for (var i = 0; i < carg; ++i)
                 {
                     var arg = node.Args.Children[i];
-                    if (func.IsLazyEvalParam(i))
+
+                    var supportColumnNamesAsIdentifiers = _features.HasFlag(Features.SupportColumnNamesAsIdentifiers);
+                    if (supportColumnNamesAsIdentifiers && func.IsIdentifierParam(i))
+                    {
+                        var identifierNode = arg.AsFirstName();
+                        Contracts.Assert(identifierNode != null);
+
+                        // Transform the identifier node as a string literal
+                        var nodeName = context.Binding.TryGetReplacedIdentName(identifierNode.Ident, out var newIdent) ? new DName(newIdent) : identifierNode.Ident.Name;
```

Comment:
>TryGetReplacedIdentName [](http://example.com/codeflow?start=55&length=23)

Why this API TryGetReplacedIdentName instead of the binding info (FirstNameInfo)?

Reply:
Same API as the one used at line 350 with FirstNameNode. I don't see what other API you'd like to call and why.